client/verta/docs/examples/tutorials.rst

```diff
@@ -1,12 +1,15 @@
 Tutorials
 =========
 
-Check out these tutorials for detailed step-by-step instructions on performing different MLOps 
+Check out these tutorials for detailed step-by-step instructions on performing different MLOps
 functions in Verta.
 
 ModelDB (Versioning and Metadata)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+* `Versioning model ingredients <tutorials/ingredients.html>`_, which allows you to have Git-like
+  for the components of models.
+
 * `Versioning a Model <tutorials/versioning.html>`_
```

Comment:
As per Manasi's comment. Recommend renaming the new page from `ingredients.html` to `versioning.html`.
```suggestion
* `Versioning a Model <tutorials/versioning.html>`_, which allows you to have Git-like
  for the components of models.
```

Reply:
Let's keep as is for now.